game/entity/tanks/types/artillery.py

```diff
@@ -0,0 +1,40 @@
+from entity.tanks.tank import Tank
+from map.hex import Hex
+
+
+class Artillery(Tank):
+    __sp: int = 1  # Speed Points
+    __dp: int = 1  # Destruction Points
+    __max_range = 3  # Manhattan max range
+    __min_range = 3  # Manhattan min range
+    __fire_deltas: tuple = Hex.fire_deltas(__min_range, __max_range)
+
+    def __init__(self, tank_id: int, tank_info: dict, colour: str, player_index: int):
+        image_path = 'game/assets/tank_classes/spg.png'
```

Comment:
image_path move to constants or other characteristics

Reply:
Will do